### PR TITLE
[PROTO-1842] Robust status report

### PR DESCRIPTION
**Pull request by phelpsdb**
Status report displayed in table format. Includes chain healthy, port 30300 availability, IP match, disk usage, db size, and debug comments.

### CODE REVIEW

File: cmd/audius-ctl/status.go [hunk 1]
```diff
@@ -2,45 +2,308 @@ package main
...
-			return nil
+			go func() {
+				wg.Wait()
+				close(resultsChan)
+			}()
+
+			var results []hcResult
+			for r := range resultsChan {
+				results = append(results, r)
+			}
```
Comment (alecsavvy) [inline cmd/audius-ctl/status.go:95]: interesting waitgroup usage

Comment (alecsavvy) [reply cmd/audius-ctl/status.go:95]: i don't think you _need_ the results channel and you can just wait the wg in the main thread, add to the results straight from the goroutine. not necessary too

**Review by alecsavvy — APPROVED** [review]
epic

File: cmd/audius-ctl/status.go (hunk 1 above)
Comment (phelpsdb) [reply cmd/audius-ctl/status.go:95]: I think I'll leave it for now, it's not thread safe to append to a slice, and while we could technically add to the results array by index, I don't think it's a necessary optimization. (https://stackoverflow.com/questions/44152988/append-not-thread-safe)

**Pull request merged**